src/main/java/com/scalar/kelpie/stats/Stats.java

```diff
@@ -38,6 +38,9 @@ public Stats(Config config) {
    */
   public void recordLatency(long latencyMillis) {
     histogram.recordValue(latencyMillis);
+    if (config.isLatencyLogEnabled()) {
+      logger.info(System.currentTimeMillis() + " " + latencyMillis);
```

Comment:
I'm wondering if we can identify latency log in case both latency log and real-time reporting are enabled.
(I'm not sure if there is such a case)
We might need to output some tag as a prefix of the log like "[latency]" to enable filtering?

Reply:
Thank you for the suggestion!
I added a tag `[Latency]`.